templates/vue/cypress/integration/node-appearance.spec.js

```diff
@@ -1,65 +1,32 @@
-import { setup } from "../support/utils"
-
 describe("Node Appearance", () => {
   beforeEach(() => {
     cy.fixture("root.json").as("oneNode")
-    setup("@oneNode")
+    cy.setup("@oneNode")
   })
 
-  const setupThenTest = (newNode, assertions) => {
+  it("should be able to edit a node's appearance using the node modal", () => {
     cy.getSelectedNode().then(node => {
-      cy.wrap(node).editNodeInStore(newNode)
-      cy.getByTestId(`node-${node.id}`).within(() => assertions(node))
-    })
-  }
+      cy.openModal("edit", node.id)
+      cy.contains(/appearance/i).click()
 
-  it(`
-    Given: A Tapestry node
-    When: An image is added then removed
-    Then: It should first show the image then hide it
-  `, () => {
-    const newNode = {
-      imageURL:
-        "https://image.shutterstock.com/z/stock-photo-colorful-flower-on-dark-tropical-foliage-nature-background-721703848.jpg",
-    }
-    setupThenTest(newNode, () => {
-      cy.get("image").should("have.attr", "href", newNode.imageURL)
-    })
-  })
+      cy.contains(/node title/i).click()
+      cy.contains(/media button/i).click()
+      cy.contains(/progress bar/i).click()
 
-  it(`
-    Given: A Tapestry node
-    When: Its title is hidden
-    Then: It should hide its title
-  `, () => {
-    const newNode = {
-      hideTitle: true,
-    }
-    setupThenTest(newNode, node => {
-      cy.getByTestId(`node-title-${node.id}`).should("not.exist")
-    })
-  })
+      cy.contains(/thumbnail/i).click()
 
-  it(`
-    Given: A Tapestry node
-    When: Its progress bar is hidden
-    Then: It should hide its progress bar
-  `, () => {
-    setupThenTest({ hideProgress: true }, node => {
-      cy.getByTestId(`node-progress-${node.id}`).should("not.exist")
-    })
-  })
+      const url =
+        "https://image.shutterstock.com/z/stock-photo-colorful-flower-on-dark-tropical-foliage-nature-background-721703848.jpg"
```

Comment:
Could we use a creative commons image (e.g. Wikimedia) and make sure the URL is reliable enough that it won't break in the future?

Reply:
Yes that sounds good!